### ISSUE TITLE
Passkey request disappears if I need to re-input master password

### ISSUE THREAD

**Issue by jtaseff-gfms** [bitwarden/clients#12513]
### Steps To Reproduce

1. Bitwarden idle for long enough that it will prompts for master password the next time you use it
2. Go to a website that has a passkey registered
3. Try to use the passkey
4. Bitwarden prompts for master password
5. Bitwarden goes to the main view without being able to use the passkey

### Expected Result

In previous versions, this would still pop up the confirmation box to use the passkey after logging in

### Actual Result

After inputting master password, Bitwarden goes to the main view without being able to use the passkey

### Operating System

Windows

### Operating System Version

Windows 11

### Web Browser

Firefox

### Build Version

2024.12.2

### Issue Tracking Info

- [X] I understand that work is tracked outside of Github. A PR will be linked to this issue should one be opened to address it, but Bitwarden doesn't use fields like "assigned", "milestone", or "project" to track progress.

**Comment by toprowtech** [comment]
I am having this trouble as well. When logging into Google Gmail (for example) with a passkey. If the extension vault is locked, the extension requests unlock via Hello, PIN or password, etc., once the vault is unlocked, the extension shows items from the vault. It does not offer the Passkey.

If the vault is unlocked, the passkey is presented correctly.

I can reproduce this behavior on two different Windows 11 Pro computers. Google Chrome with extension version 2024.12.2.

Previously to 2024.12.2, the passkey would be presented after unlocking the vault as expected.

**Comment by rcmdkick** [comment]
Hey!

I have the same annoying issue. Already written the behaviour, but on Windows 10 Pro.

**Comment by jtodddd** [comment]
Hi there,

This issue has been escalated for further investigation. If you have more information that can help us, please add it below. 

Thanks!

**Comment by toprowtech** [comment]
I went to create a screencast showing the behavior today and found that with Chrome extension 2024.12.3, the passkey behavior worked as it should.

I tested with another profile that still had extension 2024.12.2 and the passkey behavior when the vault was locked did not work correctly (it functioned as described above). However, I then updated that profile to extension 2024.12.3 and it did work correctly.

In short, this issue may have been resolved in the latest update 2024.12.3.

**Comment by closebot-bw** [comment]
⚠️ **Stale Issue Notice**

This issue has been automatically marked as stale due to inactivity. It will be closed in **2 weeks** (October 15, 2025) if no further activity occurs.

If this issue is still relevant and you would like to keep it open, please:
- Comment on this issue to show continued interest
- Provide any additional information or updates
- Confirm that the issue still exists in the latest version

Thank you for your contribution to this project! 🙏

**Comment by closebot-bw** [comment]
🔒 **Issue Closed Due to Inactivity**

This issue has been automatically closed due to lack of activity for an extended period. We periodically review and close inactive issues to help maintain our issue tracker and focus on current priorities.

**If this issue is still relevant:**
- Please create a new issue with updated information
- Include steps to reproduce the problem if it's a bug report
- Mention if this issue still occurs in the latest version

Thank you for your contribution to this project. Your feedback helps us improve! 🙏
